README.md

```diff
@@ -1,4 +1,4 @@
-# Temporal Python SDK
+![Temporal Python SDK](scripts/_img/banner.svg)
```

Comment:
Can you shrink the header height down considerably? I think it should be a quarter or less of what it currently is. There's no value in the precious vertical space wasted IMO. Yes that makes the snake smaller, but I think it's worth it if it's really a "banner".

Reply:
I didn't mean shrink the title text so small, I just meant the image size. Can leave the title text the same size it was.

Reply:
Looks great! Will make sure CI completes and merge shortly.